contribs/gmf/apps/desktop_alt/index.html.ejs

```diff
@@ -226,10 +226,10 @@
               </div>
             </div>
           </div>
-          <div ng-show="mainCtrl.drawLidarprofilePanelActive" class="row profile-panel">
+          <div ng-show="mainCtrl.drawLidarprofilePanelActive" class="row lidar-profile-panel">
             <div class="col-sm-12">
               <div class="gmf-app-tools-content-heading">
-                {{'Lidar profile'|translate}}
+                {{'Lidar profile (only on the canton of Neuchâtel)'|translate}}
```

Comment:
Why adding this mention? The tool works outside of Neuchatel, doesn't it? Or is it because the "desktop_alt" app only has lidar data in canton NE?

Reply:
It's only for our desktop_alt demo. It's not common to add info related to the config here. But desktop_alt changes is not exported in customer projects and we have only data on NE.